bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -322,38 +342,378 @@
+		<state readOnly="true" pattern="%d %%"/>
+	</channel-type>
+
+	<channel-type id="vh_relativehumidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Relative Humidity</label>
+		<description>Relative humidity exhaust air</description>
+		<state readOnly="true" pattern="%d %%"/>
+	</channel-type>
+
+	<channel-type id="vh_co2level">
+		<item-type>Number:Dimensionless</item-type>
+		<label>CO2 Level</label>
+		<description>CO2 level exhaust air</description>
+		<state readOnly="true" pattern="%d ppm"/>
+	</channel-type>
+
+	<channel-type id="vh_supplyinlettemp">
+		<item-type>Number:Temperature</item-type>
+		<label>Supply Inlet Temperature</label>
+		<description>Supply inlet temperature</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="vh_supplyoutlettemp">
+		<item-type>Number:Temperature</item-type>
+		<label>Supply Outlet Temperature</label>
+		<description>Supply outlet temperature</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="vh_exhaustinlettemp">
+		<item-type>Number:Temperature</item-type>
+		<label>Exhaust Inlet Temperature</label>
+		<description>Exhaust inlet temperature</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="vh_exhaustoutlettemp">
+		<item-type>Number:Temperature</item-type>
+		<label>Exhaust Outlet Temperature</label>
+		<description>Exhaust outlet temperature</description>
+		<state readOnly="true" pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="vh_actualexhaustfanspeed">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Actual Exhaust Fan Speed</label>
+		<description>Actual exhaust fan speed</description>
+		<state readOnly="true" pattern="%d rpm"/>
+	</channel-type>
+
+	<channel-type id="vh_actualinletfanspeed">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Actual Inlet Fan Speed</label>
+		<description>Actual inlet fan speed</description>
+		<state readOnly="true" pattern="%d rpm"/>
+	</channel-type>
+
+	<channel-type id="vh_nominalventenable">
+		<item-type>Switch</item-type>
+		<label>Nominal Ventilation Value Transfer</label>
+		<description>Nominal ventilation value transfer enabled</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="vh_nominalventrw">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Nominal Ventilation Value</label>
+		<description>Nominal ventilation value</description>
+		<state readOnly="true">
+			<options>
+				<option value="0">read-only</option>
+				<option value="1">read/write</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="vh_nominalventilationvalue">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Nominal Ventilation Value</label>
+		<description>Nominal ventilation value</description>
+		<state readOnly="true" pattern="%d %%"/>
+	</channel-type>
```

Comment:
Do these have the same label by intention?

Reply:
Not by my intention :) .. again taken from the OT specs (or rather from the OTGW source code) which I just checked and indeed has two different message codes with the same description. I don't know what they mean exactly, the percentage value seems clear but the r/w value isn't to me. Mapping every message code to a channel, I simply aim to support the full set.

Reply:
OK